sysvabi64/sysvabi64.rst

```diff
@@ -750,23 +757,26 @@ along with the assumptions that the code model may make.
   5. The text segment maximum size is limited to 2GiB by
   R_AARCH64_PLT32 relocations from ``.eh_frame`` sections.
 
-  6. While designing the code models it was estimated that only 2.6%
+  6. The distance to the GOT in the small and medium code model is
+  limited to 2GiB by the R_AARCH64_GOTPCREL32 relocations.
```

Comment:
Basically in all cases (including large model) the max distance from text to GOT is 2GB if R_AARCH64_GOTPCREL32 is used. It is 4GB otherwise as the basic range of ADRP.

So I think it doesn't really need to be added to the table since there is no difference between the code models.